app/javascript/components/supplemental_application/SupplementalApplicationHeader.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+import PageHeader from '../organisms/PageHeader'
+
+const SupplementalApplicationHeader = ({ applicationNumber, applicantName, listingId }) => {
+  const title = `${applicationNumber}: ${applicantName}`
+  const breadcrumbs = [
+    { title: 'Lease Ups', link: '/lease_ups' },
+    { title: 'Cameo Apartments', link: `/listings/${listingId}/lease_ups` },
```

Comment:
@fddayan the title should be the listing name here and the route needs to be updated to the refactored routes